src/pages/search/registration_search/RegistrationSearchBar.tsx

```diff
@@ -337,3 +340,27 @@ const SelectedFundingFacetButton = ({ fundingIdentifier }: SelectedFundingFacetB
 
   return <>{fundingSourcesQuery.isLoading ? <Skeleton sx={{ width: '7rem', ml: '0.25rem' }} /> : fundingName}</>;
 };
+
+const FilterButton = () => {
+  const { t } = useTranslation();
+  const { values, setFieldValue, submitForm } = useFormikContext<SearchTermProperties>();
+
+  return (
+    <Button
+      sx={{ mr: '0.5rem' }}
+      data-testid={dataTestId.startPage.advancedSearch.activateFilterButton}
+      startIcon={<FilterAltIcon />}
+      onClick={() => {
+        const newProp: SearchTermProperty = { fieldName: '', value: '' };
+        if (values.properties.length === 0) {
+          setFieldValue('properties', [newProp]);
+        } else {
+          setFieldValue('properties', []);
+          submitForm();
+        }
+      }}
+      variant={values.properties.length ? 'contained' : 'outlined'}>
```

Comment:
Sikker på at du ikke tenker på "disabled" istedet for variant.

Reply:
Jepp, knappen skal vise/skjule filteret. Variant contained/outlined gjør at fargen på knappen endres fra ikke-aktiv (ingen filter) til aktiv (med filter)